_docs-sources/guides/working-with-code/forking.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_label: "Forking our code"
+---
+
+# Forking the Gruntwork Infrastructure as Code Library
+
+The [Gruntwork Terms of Service](https://gruntwork.io/terms/) give you permissions to fork the code from the Gruntwork
+Infrastructure as Code Library into your own repos. This is useful if your company does not allow external dependencies (e.g., you
+have a company policy that requires all source code to be pulled from an internal GitHub Enterprise or BitBucket
+server) or if you need to make modifications to the Infrastructure as Code Library that you do not wish to contribute back to
+Gruntwork. This section will walk you through what you need to do to fork the code.
+
+The definition of an _Authorized User_ from the Gruntwork Terms of Service does NOT change if you fork the
+code. That is, if you create internal forks and give 50 users access to those internal forks, then the Gruntwork
+License requires that you pay for 50 Authorized Users.
+
+## How to fork the code
+
+Here is how you fork the code in the Gruntwork Infrastructure as Code Library:
+
+1.  Copy each Gruntwork repo into your private repositories.
+2.  You’ll also want to copy all the versioned releases (see the `/releases` page for each repo).
+3.  For repos that contain pre-built binaries (such as `ssh-grunt` mentioned earlier), you’ll want to copy those binaries
+    as well.
+4.  Within each repo, search for any cross-references to other Gruntwork repos. Most of the repos are standalone, but
+    some of the Terraform and Go code is shared across repos (e.g., the `package-kafka` and `package-zookeeper` repos
+    use the `module-asg` repo under the hood to run an Auto Scaling Group). You’ll need to update Terraform source URLs
+    and Go import statements from `github.com/gruntwork-io` to your private Git repo URLs.
+
+You’ll want to automate the entire process above and run it on a regular schedule (e.g., daily). The Gruntwork
+Infrastructure as Code Library is updated continuously, both by the Gruntwork team and contributions from our community
+of customers (see the monthly [Gruntwork Newsletter](https://blog.gruntwork.io/tagged/gruntwork-newsletter) for details),
+so you’ll want to pull in these updates as quickly as you can.
+
+## How to use your forked code
+
+Once you’ve forked the code, using it in is very similar to what is outlined in [Using Terraform Modules](#using_terraform_modules) and
+[Using scripts and binaries](#using_scripts_binaries), except for the following differences:
```

Comment:
These anchor links are broken. I think the first could possible go to [this intro section](https://deploy-preview-154--pensive-meitner-faaeee.netlify.app/docs/intro/first-deployment/using-terraform-modules), but we don't have a good answer for the latter at the moment. Thoughts?

Reply:
I wonder if the right thing to do is to allow it to be broken. My thought process:
- If we don't have a place to link it to and we link it to an existing place which doesn't really help the user understand anything then we'll have no way in the future to discover this (unless someone tells us about it)
- If we leave it broken - or update the link to a known broken thing (eg: "replace-me-when-we-have-this-section") then we'll be able to punt dealing with the problem until we want to turn on the link validator which we are about to build. At that time - perhaps we'll have a better section for this?

Reply:
Where would you propose we link to? Two candidates for this content seem to be
- Somewhere in the intro guide
- The forthcoming Tools reference section

It's weird to leave it broken, though. If we link to tools, perhaps we should have a stub there with an admonition saying this isn't done, and with a link back to the original section of the guide on Gruntwork.io? Or even a redirect? That would feel better, I think.